### PR TITLE
Size bucket filtering v2 [FX-1948]

### PR DESCRIPTION
**Pull request by pvinis**
v2 of #5670 after @sweir27 found out that the filter didn't do its job everywhere it existed.

- search (http://localhost:5000/search?term=banana)
https://drive.google.com/file/d/1-2p0UPcjZL_CPssYcFpz0h4vpb2Q_OKm/view?usp=sharing
- collect (http://localhost:5000/collect)
https://drive.google.com/file/d/1-1zhgakfkHofQOnhWDmO671lofVvqgwY/view?usp=sharing
- collection (http://localhost:5000/collection/andy-warhol-bananas)
- artist auctions (http://localhost:5000/artist/andy-warhol/auction-results)
https://drive.google.com/file/d/1-0y2u1B9sSOIpspjXo8_KuYOufE0Gnru/view?usp=sharing
- artist works-for-sale (http://localhost:5000/artist/andy-warhol/works-for-sale)
https://drive.google.com/file/d/1--tbTP5hS-Mfp_CkEyTC_oYyA9bWf6Hn/view?usp=sharing

Change the size filter ui to the one used in auctions.

Also use the newly added `sizes` to filter artworks.

example query for both the auction results and artwork results using `sizes`:
```graphql
{
  artist(id: "andy-warhol") {
    name
    internalID
    auctionResultsConnection(first: 3, sizes: [LARGE]) {
      edges {
        node {
          title
        }
      }
    }
  }
  artworksConnection(first: 3, sizes: [SMALL]) {
    edges {
      node {
        internalID
        title
        sizeScore
      }
    }
  }
}
```

### CODE REVIEW

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx [hunk 1]
```diff
@@ -196,11 +203,11 @@ export const ArtworkFilterContextProvider: React.FC<SharedArtworkFilterContextPr
     ZeroState,
 
     // Filter manipulation
-    isDefaultValue: field => {
+    isDefaultValue: (field) => {
```
Comment (damassi) [inline src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:206]: I think this needs a rebase from master to avoid the prettier churn?

Comment (pvinis) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:206]: I tried running prettier in master and about a billion files got changed, so I guess we can keep that change here and do the whole project in another PR 🤔

Comment (damassi) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:206]: With prettier v2 we had to set this to `avoid` for consistency with the rest of the codebase: https://github.com/artsy/force/blob/master/package.json#L403. Wondering why your setup isn't picking up this setting? No harm in keeping it here, was more curious about if your environment is set up correctly. 

Def plan on running over the whole codebase with Prettier v2 defaults but there's a lot of open PRs right now, was gonna wait tilil things are less noisy.

Comment (pvinis) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:206]: oh? hm I'll check the setup tomorrow.. interesting 🤔

Comment (anandaroop) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:206]: Hey @damassi @pvinis I'm going to go ahead and merge this as-is, in order to unblock #5658 — some of this churn will be undone in that PR in any case. 

Thanks in advance for taking a look at the setup tomorrow Pavlos.

**Pull request merged**